pkg/rtc/transport.go

```diff
@@ -26,7 +26,8 @@ import (
 )
 
 const (
-	negotiationFrequency = 150 * time.Millisecond
+	negotiationFrequency       = 150 * time.Millisecond
+	dtlsRetransmissionInterval = 100 * time.Millisecond
```

Comment:
Does this will cause retransmission too often? for a network with rtt > 100 ms

Reply:
Yes, it will @jie. It will retransmit every 100 ms or so. So, it could be up to 4 sends (1 original + 3 retransmit) for the longest links too (I think normally, the longest latency is about 400 ms, of course there are going to be really bad cases). But, for most cases, it should not be retransmitting. This is mainly to catch the penalty of 1 second even if 1 packet is lost.

Reply:
Sounds good. jie in github is another people :) @boks1971

Reply:
Yeah, I realised that soon after I hit enter and fixed it :-) But, that Jie probably got a weird notification :-)